simple-cdd/profiles/urna.postinst

```diff
@@ -19,4 +19,8 @@ chown -R keyboarduser /opt/keyboard-websocket
 cd /opt/keyboard-websocket
 pip install --no-index -f packages -r requirements.txt
 
+cd /srv
+wget http://downloads.eduardosan.com/eleicoes-latest.tar.gz
```

Comment:
A pergunta que fica então é: como carregar o sistema na distro?


Enviado do meu smartphone Samsung Galaxy.
-------- Mensagem original --------De: Thomaz de Oliveira dos Reis <notifications@github.com> Data: 21/05/16  13:56  (GMT-03:00) Para: thor27/debian-urna <debian-urna@noreply.github.com> Cc: "Eduardo F. Santos" <eduardo@eduardosan.com>, Author <author@noreply.github.com> Assunto: Re: [thor27/debian-urna] altera CD para iniciar o sistema da Urna na página inicial (#1) 
In simple-cdd/profiles/urna.postinst:
> @@ -19,4 +19,8 @@ chown -R keyboarduser /opt/keyboard-websocket
>  cd /opt/keyboard-websocket
>  pip install --no-index -f packages -r requirements.txt
>  
> +cd /srv
> +wget http://downloads.eduardosan.com/eleicoes-latest.tar.gz

Então, não fica legal esse wget aqui.. ele poderia ser feito no script update.sh que "monta" os dados para iso e ser copiado para /opt da pasta root por exemplo.

Caso seja interessante ter update com a "urna rodando" poderia colocar o update no terminal de configuração [1]. Agora prestar atenção a configuração da internet pq por default fica desativado na urna. 

[1] https://github.com/thor27/debian-urna/blob/master/root/usr/local/bin/config

de qualquer forma no terminal de configuração existe um terminal root. só acessar ele e pode fazer o que quiser.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub